app/lib/common/widgets/emoji_picker_widget.dart

```diff
@@ -26,6 +30,33 @@ class EmojiPickerWidget extends StatelessWidget {
         size == null ? MediaQuery.of(context).size.width : size!.width;
     final cols = min(width / (EmojiConfig.emojiSizeMax * 2), 12).floor();
 
+    final emojiConfig = EmojiViewConfig(
+      backgroundColor: Theme.of(context).colorScheme.surface,
+      columns: cols,
+      emojiSizeMax: EmojiConfig.emojiSizeMax,
+    );
+    final catConfig = CategoryViewConfig(
+      customCategoryView: (config, state, tab, page) =>
+          actionBar(context, emojiConfig, state, tab, page),
+    );
```

Comment:
So we have added custom view for category? Not quick getting how it was working when there was no `actionBar` on prior code?

Reply:
well, it wasn't working. You couldn't close the emoji-picker on desktop.  that's what this PR fixes.

The default category view only supports either having a backspace or a search button. and while the action-bar-provides both action, having two bar is just very ugly and distracting.

that's why I replaced the category view with our own version including all features - search, backspace and close - and disabled the actionbar.